### PR TITLE
Machine Pool creation: add example

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

File: examples/create_machine_pool/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Creating a Machine Pool for existing Openshift cluster
+
+This Terraform example creates a Node Pool for existing OCM cluster. Creating new Machine pool allows users to add different machine types in the cluster..
```
Comment (amalykhi) [inline examples/create_machine_pool/README.md:3]: `This Terraform example creates a **Machine Pool** for existing OCM cluster. Creating new Machine pool allows users to add different machine types in the cluster.`

File: examples/create_machine_pool/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        ```
+        export TF_VAR_name=<name>
+        ```
+    5. This value sets the AWS instance type used for instances created in this Machine Pool
+        ```
+        export TF_VAR_machine_type=<machine_type>
+        ```
+    6. This value sets the amount of the machine created in this machine pool
+        ```
+        export TF_VAR_replica=<replica>
```
Comment (amalykhi) [inline examples/create_machine_pool/README.md:44]: replica**s**

File: examples/create_machine_pool/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* You have already created OCM cluster.
+
+
+## ROSA cluster creation
+
+1. To run the `terraform apply` you need to set up some variables. This guide uses environmental variables. For more on Terraform variables, see [Managing Variables](https://developer.hashicorp.com/terraform/enterprise/workspaces/variables/managing-variables) in the Terraform documentation.
+
+   > **NOTE**: If you exported these variables in your current command-line session when running the account-roles Terraform example, you do not need to export them again.
+
+   Run the following commands to export your variables. Provide your values in lieu of the brackets. Note that any values declared in the `variables.tf` are the default values if you do not export a superseding value.
```
Comment (amalykhi) [inline examples/create_machine_pool/README.md:20]: Do you want to mention here other possible variables? or link the [terraform.io](https://registry.terraform.io/providers/terraform-redhat/ocm/latest/docs/resources/machine_pool) page?

**Pull request merged**